itests/retry_test.go

```diff
@@ -0,0 +1,22 @@
+package itests
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"golang.org/x/xerrors"
+
+	"github.com/filecoin-project/go-jsonrpc"
+
+	"github.com/filecoin-project/lotus/lib/retry"
+)
+
+func TestRetryErrorIsInTrue(t *testing.T) {
```

Comment:
`itests/` is really just for integration tests which require running test nodes. This test can be in `lib/retry/retry_test.go`; CI will still run that with other unit tests